datasets/ru/nsd_isin/crawler.py

```diff
@@ -87,7 +80,7 @@ def crawl_item(context: Context, url: str):
         issuer.id = context.make_id(isin_code, issuer_name)
     issuer.add("country", "ru")
     for prop, prop_val in values["issuer"].items():
-        issuer.add(prop, prop_val)
+        issuer.add(prop, prop_val, lang="rus")
```

Comment:
We have `data.lang = rus` in the dataset metadata, so in the crawler itself we don't really need to override, I think :)